modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItTablesApiTest.java

```diff
@@ -255,21 +256,44 @@ public void testGetTableFromLaggedNode() {
      */
     @Test
     public void testAddIndex() {
-        clusterNodes.forEach(ign -> assertNull(ign.tables().table(TABLE_NAME)));
+        Ignite ignite = clusterNodes.get(0);
 
-        Ignite ignite0 = clusterNodes.get(0);
+        createTable(ignite, TABLE_NAME);
 
-        createTable(ignite0, TABLE_NAME);
+        String indexName = "TEST_INDEX";
```

Comment:
Let's make it constant as it done for "TABLE_NAME".

Reply:
fix it